ios/BT/Extensions/Exposure Notifications/Scoring.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import ExposureNotification
+
+extension ExposureManager {
+
+  private static let matchedKeyCountMax = 3
+
+  static func score(summary: ENExposureDetectionSummary,
+                    with configuration: ExposureConfiguration) -> Bool {
+    let matchedKeyCount = Int(summary.matchedKeyCount)
+    if matchedKeyCount == 0 {
+      return false
+    }
+    if matchedKeyCount <= matchedKeyCountMax {
+      // If the weighted average duration per matched key exceeds the
+      // threshold, we know that there was at least one day where the exposure
+      // exceeded the threshold.
+      let weightedDurationForKeys = weightedDuration(summary: summary,
+                                              config: configuration) / matchedKeyCount
+      if (weightedDurationForKeys >= configuration.triggerThresholdWeightedDuration * 60) {
```

Comment:
```suggestion
     let  isAboveAverageDurationThreshold = (weightedDurationForKeys >= configuration.triggerThresholdWeightedDuration * 60)
     return isAboveAverageDurationThreshold
```

Reply:
not sure if this is valid swift, but the
```
if (expression) 
  return true
else
  return false
```

seems odd to my eye.

Reply:
yes, its better, thanks